pkgs/applications/misc/gsctl/default.nix

```diff
@@ -11,13 +11,19 @@ buildGoModule rec {
     sha256 = "sha256-uCNWgaLZMm1vPxFduj8mpjKYuYlp1ChF6bK+bmAWy50=";
   };
 
+  postPatch = ''
```

Comment:
It would be nice to add some comments to know why those tests failed.

Reply:
Done